site/diagrams/sequenceDiagrams/protocolAdapterDeviceAuthentication_Success.txt

```diff
@@ -0,0 +1,19 @@
+title Protocol Adapter Device Authentication (success)
+
+Device->+Protocol Adapter: authenticate(type, auth-id, secrets)
+note over Device:
+Denotes the process of authentication
+(that differs for every protocol)
+end note
+
+Protocol Adapter->+Credentials Service: transfer("credentials/TENANT", "get", type, auth-id)
+note over Credentials Service: lookup credentials
+Credentials Service->-Protocol Adapter: transfer("credentials/TENANT", "200", credentials)
+
+Protocol Adapter->Protocol Adapter: verify(credentials, secrets)
```

Comment:
this is not necessary IMHO, the note is sufficient to illustrate what the protocol adapter is supposed to do with the retrieved credentials